src/sonic-yang-models/yang-models/sonic-fine-grained-ecmp.yang

```diff
@@ -48,12 +52,33 @@ module sonic-fine-grained-ecmp {
                  "Fine grained ecmp is used when the nexthop IPs match the FG_NHG_MEMBER IPs";
             }
 
+            enum prefix-based {
+                description
```

Comment:
should we update the description of route-based to say that it matches on prefix AND nexthops, otherwise may be confusing to read the description of prefix-based, as it is the same as route-based in the description.

Reply:
Updated as requested.